src/components/Calculator.js

```diff
@@ -246,6 +215,9 @@ class Calculator extends Component {
               </Flex>
             </Flex>
           </Flex>
+          <Text textAlign="left" mb={1} className={css({marginBottom: 0, paddingTop: 20, paddingLeft: 3, fontSize: 12, color: 'grey'})}>
```

Comment:
I think `SmallText` will look a bit better here

Reply:
I tried using it, I wasn't able to change the fontsize

Reply:
Without changing the fontSize for all the smallText components anyway, that might just be my unfamiliarity with react. Antton wanted the text to be the this size.

Reply:
OK cool. Well in that case you probably want to use `BodyText`. `Text` is a base component used underneath `SmallText` and `BodyText` and isn't (or at least shouldn't) be used anywhere, so if we use it here it'll be an outlier aesthetically and in the code.

Reply:
ok